tests/generation/pdp/test_raw_course.py

```diff
@@ -13,10 +13,20 @@
     ["normalize_col_names"],
     [(False,), (True,)],
 )
-def test_raw_cohort_record(normalize_col_names):
+def test_raw_course_record(normalize_col_names):
```

Comment:
Looks like the original test name was a copy-paste error, yeah?

Reply:
Ya that's what I'm guessing